ui/package-lock.json

```diff
@@ -21923,12 +21897,6 @@
         }
       }
     },
-    "eslint-visitor-keys": {
-      "version": "1.3.0",
-      "resolved": "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-1.3.0.tgz",
-      "integrity": "sha512-6J72N8UNa462wa/KFODt/PJ3IU60SDpC3QXC1Hjc1BXXpfL2C9R5+AU7jhe0F6GREqVMh4Juu+NY7xn+6dipUQ==",
-      "dev": true
-    },
     "eslint-webpack-plugin": {
       "version": "3.1.1",
       "resolved": "https://registry.npmjs.org/eslint-webpack-plugin/-/eslint-webpack-plugin-3.1.1.tgz",
```

Comment:
Can you check why this has been updated?

Reply:
@DeRaowl  Changed to parser: "@babel/eslint-parser" from  parser: "@babel/eslint". For this I've added the dependency in package.json.  Hence the update in package-lock.json